application/src/test/java/org/togetherjava/tjbot/features/mathcommands/TeXCommandTest.java

```diff
@@ -14,7 +14,9 @@
 import java.util.ArrayList;
 import java.util.List;
 
-import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.ArgumentMatchers.argThat;
+import static org.mockito.ArgumentMatchers.contains;
+import static org.mockito.ArgumentMatchers.startsWith;
```

Comment:
Not sure about mockito imports tbh, I would understand if we keep star imports here. Though that is not really consistent. Same with assertions in other files.

Reply:
I'd do it like this for consistency's sake.